app/channels/tims_notifications_channel.rb

```diff
@@ -0,0 +1,21 @@
+class TimsNotificationsChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from "tims_notifications_channel"
+  end
+
+  def unsubscribed
+    # Any cleanup needed when channel is unsubscribed
+  end
+
+  def receive(data)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `data`. If it's necessary, use `_` or `_data` as an argument name to indicate that it won't be used. You can also write as `receive(*)` if you want the method to accept any arguments but don't care about them.